internal/runtime/catalog/catalog.go

```diff
@@ -330,6 +330,9 @@ var emptyGroupVersionKind = schema.GroupVersionKind{}
 
 // GVHToPath calculates the path for a given GroupVersionHook.
 // TODO: discuss eventually in which package this should be placed.
-func GVHToPath(gvs GroupVersionHook) string {
-	return fmt.Sprintf("/%s/%s/%s", gvs.Group, gvs.Version, gvs.Hook)
+func GVHToPath(gvh GroupVersionHook, name string) string {
```

Comment:
Can you please adjust the other usages of this func otherwise we run into compile errors